readthedocs/templates/builds/build_detail.html

```diff
@@ -161,18 +161,20 @@
           </p>
         </div>
       {% endif %}
-      {% if build.finished and not build.using_latest_config %}
+
+      {# This message is not dynamic and only appears when loading the page after the build has finished #}
+      {% if build.finished and build.deprecated_config_used %}
         <div class="build-ideas">
           <p>
             {% blocktrans trimmed with config_file_link="https://docs.readthedocs.io/page/config-file/v2.html" %}
-              <strong>Configure your documentation builds!</strong>
-              Adding a <a href="{{ config_file_link }}">.readthedocs.yaml</a> file to your project
-              is the recommended way to configure your documentation builds.
-              You can declare dependencies, set up submodules, and many other great features.
+              <strong>Your builds will stop working soon!</strong><br/>
+              Building without a configuration file (or using v1) is deprecated and will be removed soon.
```

Comment:
I'm not sold on mentioning v1 configuration here, it will likely confuse more users than are using v1 in their projects. Same note as above here either way, this could describe the effect to the user, rather than describe the change we're making.

```suggestion
              Configuration files will <strong>soon be required</strong> by projects, and will no longer be optional.
```

If there isn't a clear way to also mention a v1 config, I think it's plenty to add warnings in our docs for this case.

Reply:
With this modification, people using a v1 config file would ignore this message, because they are using a configuration file already. Don't you think? I'm doing adding `(v2)` to your suggestion to make this clear.

```suggestion
              Configuration files (v2) will <strong>soon be required</strong> by projects, and will no longer be optional.```

Reply:
I changed my mind. We don't have too many people using `v1` and I think that mentioning v2 may only add noise here.

Reply:
Aye, I should have mentioned I was thinking of that v1 metric that you noted, where only ~40 projects used v1 in the last month or something. We ensure we talk to these users in a notification email/etc though.